src/ComposedModal/ComposedModal.svelte

```diff
@@ -67,7 +67,7 @@
     if (selectorPrimaryFocus == null) return;
     const node =
       (element || innerModal)?.querySelector(selectorPrimaryFocus) || buttonRef;
-    if (node != null) node.focus();
+    if (node !== null) node.focus();
```

Comment:
Line above already has some optional chaining, could be used here as well.
```suggestion
    node?.focus();
```